### ISSUE TITLE
Feature request: Repeat last command

### ISSUE THREAD

**Issue by benjamingorman** [platformio/platformio-atom-ide-terminal#171]
Often while developing it is necessary to repeat a certain shell command multiple times. For example while testing code I am frequently running the command "go test" to run a test suite.
One way to handle this would be use the existing insert-custom-text commands. But adjusting the text for these is inconvenient if you are repeating a variety of different commands.

The easiest way I have found to repeat an arbitrary command with the plugin is:

- Hit ctrl-` twice to to switch the cursor's focus to the terminal (assuming the cursor is already in a text pane)
- Press the up arrow key to get the last command from the shell's history.
- Press enter to run the command

A much nicer way would be:
- Press one key which get's the last command from the shell's history and runs it

This could be implemented as a platformio command bound to a key by the user.
Perhaps certain 'dangerous' commands such as "rm" could require confirmation before running again in order to prevent accidental mistakes.

**Comment by vmasrani** [comment]
+1 - This feature would be very helpful.

**Comment by fsramalho** [comment]
Indeed, this would be great, I would be only using platformio instead of Script :)
